.travis/runtox

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ex
+
+SCRIPTDIR=$(dirname $0)
+CONFIG=${SCRIPTDIR}/config.sh
+
+if [[ -f ${CONFIG} ]]; then
+  . ${CONFIG}
+fi
+
+tox "$@"
+if [[ "x${LSR_MSCENARIOS}" != "x" ]]; then
```

Comment:
AFAICS, the if condition is not needed since the for loop will not do anything if `$LSR_MSCENARIOS `is not defined.